interpreter/src/main/java/org/qbicc/interpreter/impl/VmImpl.java

```diff
@@ -266,6 +269,9 @@
             }
             return clazz;
         });
+
+        bootstrapComplete = true;
+        throwableClass.initializeConstantStaticFields(); // Has constant String fields that can't be initialized when we first process the class
```

Comment:
Ok, so we've got the start of the loop over loaded classes to fix this up here.  We can do the explicit list if it's easy enough to detect the classes that need it.

Reply:
Yeah, it was easy to detect.  I didn't leave the printf in initializeConstantStaticFields(), but that's how I determined that Throwable was the only one that needed it done.